indico/client/client.py

```diff
@@ -1,11 +1,19 @@
 # -*- coding: utf-8 -*-
-
+import concurrent.futures
 from typing import Union
-import urllib3
 
+from indico.client.request import (
+    GraphQLRequest,
+    HTTPRequest,
+    PagedRequest,
+    RequestChain,
+)
 from indico.config import IndicoConfig
 from indico.http.client import HTTPClient
-from indico.client.request import HTTPRequest, RequestChain, PagedRequest, GraphQLRequest
+
+import urllib3
+
+THREAD_POOL = concurrent.futures.ThreadPoolExecutor(max_workers=8)
```

Comment:
typing and you might need to isort this. Is there a pre-commit for this?

Reply:
no pre-commit